api/src/routes.php

```diff
@@ -17,33 +19,65 @@
     $this->post('/login', LoginController::class . ':login')
         ->add(new ValidationErrorResponseBuilder())
         ->add(new Validation(
-            $this->getContainer()['validation']['login'],
-            $this->getContainer()['validation']['loginTranslation']
+            $container['validation']['login'],
+            $container['validation']['loginTranslation']
         ))
-        ->add(new AuthenticationMiddleware($container, \JMP\Utils\PermissionLevel::OPEN));
+        ->add(new AuthenticationMiddleware($container, PermissionLevel::OPEN));
 
     $this->get('/events', EventsController::class . ':listEvents')
         ->add(new ValidationErrorResponseBuilder())
-        ->add(new Validation($this->getContainer()['validation']['listEvents']))
-        ->add(new AuthenticationMiddleware($container, \JMP\Utils\PermissionLevel::USER))
+        ->add(new Validation($container['validation']['listEvents']))
+        ->add(new AuthenticationMiddleware($container, PermissionLevel::USER))
         ->add($jwtMiddleware);
 
-    $this->get('/events/{id}', EventsController::class . ':getEventById')
+    $this->get('/events/{id:[0-9]+}', EventsController::class . ':getEventById')
```

Comment:
Is this regex useful or should we move all validation into validation.php? Update all routes with a such regex, where possible, e.g. /registration/eventId/userId route